docs/gettingstarted/structuring.md

```diff
@@ -1,54 +1,83 @@
 ====================
 
-We'll look at how to organize your mod into different files and what those files should do.
+In this section, we'll look at the general convention on how to organize your mod into different packages, and what those packages should do.
 
 Packaging
 ---------
 
-Pick a unique package name. If you own a URL associated with your project, you can use it as your top level package. For example if you own "example.com", you may use `com.example` as your top level package.
+Your mod should have a unique package name to avoid conflicts with anything else that might be loaded in from a Classloader. If you own a domain you would like to associate or have a domain associated with other projects already, you can use it as your **top level package**. For example, if you own the domain *"examplemod.com"*, you can choose to use `com.examplemod` as your top level package.
 
-!!! important
+**HOWEVER**
 
-    If you do not own a domain, do not use it for your top level package. It is perfectly acceptable to start your package with anything, such as your name/nickname, or the name of the mod.
+If you do not own a domain, **do *NOT* use it for your top level package!**  
+It is perfectly acceptable to start your package with anything, such as your name/nickname, or the name of the mod. For example, `username.mods` would be a good top level package. 
 
-After the top level package (if you have one) you append a unique name for your mod, such as `examplemod`. In our case it will end up as `com.example.examplemod`.
+After you pick a top level package, you should append a unique name to it for your mod. Some good examples would be `com.examplemod.awesomesauce` or `username.mods.awesomesauce`, where **AwesomeSauce** is the name of our mod.
 
 The Mod File
 ------------
 
-Generally, we'll start with a file named after your mod, and put into your package. This is the *entry point* to your mod
-and will contain some special indicators marking it as such.
+Generally, we'll start with a file named after your mod, and put into your package. This is the *entry point* to your mod and will contain some special annotations marking it as such. These annotations are `@Mod`, `@Instance`, and `@EventHandler`. Let's take a look at this example:
+
+    package com.examplemod.awesomesauce;
+        
+    import net.minecraft.init.Blocks;
+    import net.minecraftforge.fml.common.Mod;
+    import net.minecraftforge.fml.common.Mod.EventHandler;
+    import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+        
+    @Mod(modid = "awesomesauce", name = "AwesomeSauce", version = "0.1")
+    public class AwesomeSauceMod
+    {
+        @Instance
+        public static AwesomeSauceMod instance = new AwesomeSauceMod();
+        
+        @EventHandler
+        public void init(FMLInitializationEvent event)
+        {
+            // some example code
+            System.out.println("DIRT BLOCK >> "+Blocks.DIRT.getUnlocalizedName());
+        }
+    }
```

Comment:
I mean, yea, I did take it from the MDK, but having at bare bones example in documentation for people that don't have the MDK configured yet and they are looking at the documentation before diving in should at least be considered. I also find it hard to follow documentation if you have to follow a link back and forth from code and the docs. I'll add a link to the example mod on the github and remove the code block if nessasary